addons/website_sale_wishlist/static/src/js/website_sale_wishlist.js

```diff
@@ -75,10 +75,7 @@ publicWidget.registry.ProductWishlist = publicWidget.Widget.extend(VariantMixin,
         var self = this;
         var productID = $el.data('product-product-id');
         if ($el.hasClass('o_add_wishlist_dyn')) {
-            productID = $el.parent().find('.product_id').val();
-            if (!productID) { // case List View Variants
-                productID = $el.parent().find('input:checked').first().val();
-            }
+            productID = $el.parent().parent().find('.product_id').val();
```

Comment:
Since the class `o_add_wishlist_dyn` is used in two distinct buttons, I wonder if it wouldn't be better to target a more precise element than `parent().parent()` and rather use `closest(.X)`.

Also, in this case, I wonder if we cannot go through the `_getProductId($parent)` from `VariantMixin` ? (not sure about it tbh)

Reply:
I can change it to use `_getProductId()` but in the end the code is the same, the only issue currently is that using a single `.parent()` does not go back as far as needed (I honestly don't know if it's because the way the html is rendered was change between version or it was always like that)
As for the different version, it was already taken into account with the `.val()` which in the case of variant list select the checked value and in the configurator view the html only render 1 findable element to `.find(".product_id")`